wordpress-mobile/WordPress-iOS
# Updated copy for preview unavailable screen

Fixes #11943 

To test:
1. login with a self hosted site
2. go to published post
3. make changes
4. tap preview
* See the copy: "preview unavailable"

Before:
![IMG_4442](https://user-images.githubusercontent.com/1335657/59711861-f1131080-91d9-11e9-992e-a52524289a21.JPG)

After:
![IMG_4441](https://user-images.githubusercontent.com/1335657/59711868-f6705b00-91d9-11e9-82bb-90c84591b06d.JPG)
 

Update release notes:
